static.config.json

```diff
@@ -44,7 +44,10 @@
       "client": "eed-PORTAL-ENV-serverless-client",
       "lambda": "eed-PORTAL-ENV-serverless-lambda"
     },
-    "disableEddDownload": "false"
+    "disableEddDownload": "false",
+    "macOsEDDExecutableSize":130,
```

Comment:
In the event that we need to change these files, it would be better to change them in bamboo, not in a PR. These values need to also be set in deploy-bamboo.sh

Also minor nitpick, that value is the size of the download, not the executable

Reply:
Set the values according to what was done by `disableEddDownload` renamed the values to `<OS_NAME>DownloadSize`